pkg/kubectl/cmd/create_configmap.go

```diff
@@ -55,7 +60,7 @@ var (
 // ConfigMap is a command to ease creating ConfigMaps.
 func NewCmdCreateConfigMap(f *cmdutil.Factory, cmdOut io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "configmap NAME [--from-file=[key=]source] [--from-literal=key1=value1] [--dry-run]",
+		Use:     "configmap NAME [--from-file=[key=]source] [--from-binary-file=[key=]source] [--from-literal=key1=value1] [--dry-run]",
```

Comment:
It is easy programmatically to content sniff the file. I'm saying that makes it hard to reason about the ConfigMap that will be produced, even if you knew the file was supposed to be treated as a binary file